### ISSUE TITLE
Data duplication (maybe) - Plugin Settings

### ISSUE THREAD

**Issue by rahul286** [rtCamp/nginx-helper#64]
Report from a user:

I noticed both options related to nginx plugin in options table within the db have the same content: 

`rt_wp_nginx_helper_options`

and

`rt_wp_nginx_helper_global_options`

carry:

```
a:17:{s:9:"log_level";s:4:"INFO";s:12:"log_filesize";i:5;s:12:"enable_purge";i:0;s:10:"enable_map";i:0;s:10:"enable_log";i:0;s:12:"enable_stamp";i:0;s:21:"purge_homepage_on_new";i:1;s:22:"purge_homepage_on_edit";i:1;s:21:"purge_homepage_on_del";i:1;s:20:"purge_archive_on_new";i:1;s:21:"purge_archive_on_edit";i:1;s:20:"purge_archive_on_del";i:1;s:28:"purge_archive_on_new_comment";i:0;s:32:"purge_archive_on_deleted_comment";i:0;s:17:"purge_page_on_mod";i:1;s:25:"purge_page_on_new_comment";i:1;s:29:"purge_page_on_deleted_comment";i:1;}
```

Is this intentional? Or is one of them left form older version? Please let me know so I remove the duplicate if it possible, and should they stay autoloaded in the db head?

I think we created global option for multisite networkwide settings. But this options should not be set/used on single-site wordpress.

**Comment by chandrapatel** [comment]
I've explored the codebase and found that `rt_wp_nginx_helper_options` only used. I couldn't see any use of `rt_wp_nginx_helper_global_options`. This option used to save future post data by few functions but that also not used anywhere. Maybe redundant code.

In multisite, we store `rt_wp_nginx_helper_options` option in site-meta and in all subsites options table. But we only use this option from site-meta and no use of this option stored in subsites options table.

I think we need to refactor the settings page codebase.

**Comment by chandrapatel** [comment]
This has been fixed in version 2.0.0 so closing it.
